library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -316,11 +314,11 @@ data class Conversations(
         if (!KeyUtil.validateConsentSignature(signature, client.address, peerAddress, timestamp)) {
             return
         }
-        val contacts = client.contacts
-        contacts.refreshConsentList()
-        if (contacts.consentList.state(peerAddress) == ConsentState.UNKNOWN) {
-            contacts.allow(listOf(peerAddress))
-        }
+//        val contacts = client.preferences
+//        contacts.refreshConsentList()
+//        if (contacts.consentList.state(peerAddress) == ConsentState.UNKNOWN) {
+//            contacts.allow(listOf(peerAddress))
+//        }
```

Comment:
Do we want to remove these lines?

Reply:
Good call this should be handled in libxmtp now.